pkgs/by-name/we/weevely/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, python3
+, python3Packages
+, makeWrapper
+, installShellFiles
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "weevely";
+  version = "4.0.1-unstable-2023-06-18";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "epinna";
+    repo = "weevely3";
+    rev = "445bd88be1a7ae2a1aaa34b88a38a92786e56b84";
```

Comment:
Maybe you should use the git tag `v4.0.2`, which makes it easier to read, see: https://github.com/epinna/weevely3/tree/v4.0.2

The commit rev you put in is neither version 4.0.1 nor 4.0.2, it's something in between them...

Reply:
I will use the latest commit.

Reply:
If you really need this version (which is 4 commit ahead of the last release), you should maybe ask epinna to create a new relase...

Reply:
For just 4 commits, it is not needed to create a new release.

Reply:
Then I guess the features aren't that important :)